### ISSUE TITLE
HOW TO... How to stop soft coded subtitle to be shown in edit video along with the SRT subtitle

### ISSUE THREAD

**Issue by Cyberyoda1411** [SubtitleEdit/subtitleedit#8842]
When we have a soft-coded subtitle in the video, it will be shown in the video preview in SE. When we load SRT, we will have two subtitles. One is soft-coded, and the other is from the SRT file. 

Can we prevent soft-coded subtitles from being shown in the preview window in SE? 

![Soft SRT](https://github.com/user-attachments/assets/43a88556-72ec-467a-99d2-306e72f8c0c8)

**Comment by niksedk** [comment]
You're using directshow - you can probably control subtitles via some directshow control panel. 
SE is not display those subtitles on the video.

**Comment by Cyberyoda1411** [comment]
Yes, I use directshow and I didn't have any problem with these double titles before, soft-coded titles didn't show before. There is nothing in SE to set directshow, it must be you refer to some external control panel, I use a K-lite codec pack, but it is like a spaceship, they offer 10 million options for tweaking directshow, but there is nothing about the soft-code, but tons and tons of filters and options. Any idea?

**Comment by niksedk** [comment]
Sorry, I have not used the K-lite codec pack, so I do not know where to turn subs on/off.

You change change player to e.g. mpv.

**Comment by Cyberyoda1411** [comment]
Solution. I deleted K-lite codec pack and installed the LAV filters codec pack. There are no problems at all, I lost the soft-coded subtitle in SE. 
In case that it doesn't work for somebody, he must go to Lav splitter configuration and make some changes there, though I didn't need to to anything. So, I don't need to change directSthow player. For now, because I read that it will be abandoned in some future versions of Windows. But, we are all safe in the next 2 years. Until Windows 12.
